### ISSUE TITLE
Module [OutfitTransformer] is missing the required "forward" function

### ISSUE THREAD

**Issue by Anupito** [owj0421/OutfitTransformer#4]
Since you removed the "forward" and "cp_forward" functions after the CIR update, a Module [OutfitTransformer] is missing the required "forward" function error appears during evaluation. Seems like you included the necessary code directly to define logits, which is missing from the evaluation functions. Adding the new code onto the two functions solves this issue. 

<img width="1118" alt="Screenshot 2024-02-29 at 11 12 26" src="https://github.com/owj0421/OutfitTransformer/assets/155425471/173d6512-97f4-492e-9f6b-fd267c9a8fa7">
<img width="1118" alt="Screenshot 2024-02-29 at 11 13 29" src="https://github.com/owj0421/OutfitTransformer/assets/155425471/b384ec7c-d8a7-4749-ac75-fcefc2d6be69">

**Comment by bigohofone** [comment]
Thank you for letting me know. I will revise it as soon as possible and upload it with cir forward and evaluation code.

네이버 메일앱에서 보냈습니다.
-----Original Message-----
보낸사람: ***@***.***>
받는사람: ***@***.***>
참조: ***@***.***>
날짜: 2024. 2. 29 오후 7:06:01
제목: [owj0421/OutfitTransformer] Module [OutfitTransformer] is missing the required "forward" function (Issue #4)

Since you removed the "forward" and "cp_forward" functions after the CIR update, a Module [OutfitTransformer] is missing the required "forward" function error appears during evaluation. Seems like you included the necessary code directly to define logits, which is missing from the evaluation functions. Adding the new code onto the two functions solves this issue.
Screenshot.2024-02-29.at.11.12.26.png (view on web) Screenshot.2024-02-29.at.11.13.29.png (view on web)
